lib/vdc/BOVCollection.cpp

```diff
@@ -0,0 +1,529 @@
+#include <vector>
+#include <algorithm>
+#include <map>
+#include <sstream>
+#include <iomanip>
+#include <iostream>
+#include <fstream>
+#include <type_traits>
+#include "vapor/VAssert.h"
+#include <stdio.h>
+
+#ifdef _WINDOWS
+    #define _USE_MATH_DEFINES
+    #pragma warning(disable : 4251 4100)
+#endif
+#include <cmath>
+
+#include <vapor/BOVCollection.h>
+
+using namespace VAPoR;
+
+const std::string BOVCollection::_timeToken = "TIME";
+const std::string BOVCollection::_dataFileToken = "DATA_FILE";
+const std::string BOVCollection::_gridSizeToken = "DATA_SIZE";
+const std::string BOVCollection::_formatToken = "DATA_FORMAT";
+const std::string BOVCollection::_variableToken = "VARIABLE";
+const std::string BOVCollection::_endianToken = "DATA_ENDIAN";
+const std::string BOVCollection::_centeringToken = "CENTERING";
+const std::string BOVCollection::_originToken = "BRICK_ORIGIN";
+const std::string BOVCollection::_brickSizeToken = "BRICK_SIZE";
+const std::string BOVCollection::_offsetToken = "BYTE_OFFSET";
+const std::string BOVCollection::_divideBrickToken = "DIVIDE_BRICK";
+const std::string BOVCollection::_dataBrickletsToken = "DATA_BRICKLETS";
+const std::string BOVCollection::_dataComponentsToken = "DATA_COMPONENTS";
```

Comment:
I believe making the token names follow variable naming conventions makes the code less readable. For example in `_findToken(_divideBrickToken, line, _divideBrick);` you cannot tell what is happening without knowing that the variables that end in token are the static tokens whereas `_findToken(TOKEN_DIVIDE_BRICK, line, _divideBrick);` is much clearer.

Reply:
Fixed.